backend/src/main/java/gov/cdc/usds/simplereport/config/SecurityConfiguration.java

```diff
@@ -48,6 +48,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 .antMatchers("/").permitAll()
                 .antMatchers(HttpMethod.OPTIONS, "/**").permitAll()
+				.antMatchers(HttpMethod.GET, "/actuator/health").permitAll()
```

Comment:
Should use the `EndpointMatcher` so it stays working if somebody decides to move it to `/management` in the future. (Also should probably use the same indentation mechanism that the code around it uses, though editor style continues to be a thing we don't have a consistent approach to.)